apps/tailwind-components/components/BreadCrumbs.vue

```diff
@@ -14,38 +14,48 @@ withDefaults(
 </script>
 
 <template>
-  <div class="flex justify-between xl:hidden text-menu">
-    <NuxtLink :to="Object.values(crumbs).slice(-1)[0]">
-      <span class="sr-only">Go up one level</span>
-      <BaseIcon name="arrow-left" />
-    </NuxtLink>
-    <!-- <a href="#">
-      <span class="sr-only">Favorite</span>
-      <BaseIcon name="star" />
-    </a> -->
-  </div>
-  <nav
-    class="items-center hidden gap-3 tracking-widest xl:flex font-display text-heading-lg"
-    :class="{ 'justify-center': align === 'center' }"
-  >
-    <template v-for="(url, label, index) in crumbs" :key="label">
-      <NuxtLink :to="url" class="text-breadcrumb hover:underline">{{
-        label
-      }}</NuxtLink>
-      <span
-        v-if="index < Object.keys(crumbs).length - 1"
-        class="text-breadcrumb-arrow"
+  <nav aria-label="breadcrumb">
+    <ol class="flex justify-between xl:hidden text-menu">
```

Comment:
I need to do some more research on the proper way to markup multiple breadcrumbs. I think this is fine for now.